packages/react/src/components/TeachingBubble/TeachingBubble.styles.ts

```diff
@@ -254,8 +264,10 @@ export const getStyles = (props: ITeachingBubbleStyleProps): ITeachingBubbleStyl
             },
           ],
           '&:hover, &:focus': {
-            backgroundColor: palette.themeDarkAlt,
-            borderColor: palette.white,
+            backgroundColor: palette.themeDark,
+            border: `2px solid ${palette.black}`,
+            outline: `1px solid ${palette.white}`,
+            outlineOffset: '-3px',
```

Comment:
Same comment as earlier -- we should split out the `:focus` style to separately style the border & outline, and the border should be 1px.

Reply:
I've separated styling for `:hover` and `:focus` for this button and kept the :hover style for how it originally was